src/meta/play-meta.js

```diff
@@ -75,6 +76,18 @@ export const plays = [
     tags: 'JSX, State, Props',
     github: 'murtuzaalisurti',
     featured: true
```

Comment:
@Abhishek-90 There is a merge conflict in this file. Could you please resolve?

Reply:
Conflict Resolved